android-fast-dump/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# For more information about using CMake with Android Studio, read the
+# documentation: https://d.android.com/studio/projects/add-native-code.html
+
+set(TARGET fast-dump)
+project(${TARGET})
+
+# Sets the minimum version of CMake required to build the native library.
+cmake_minimum_required(VERSION 3.4.1)
+
```

Comment:
It might be a good idea to set the language standard (let's shoot for C++11) by adding:

    set(CMAKE_CXX_STANDARD 11)

Reply:
done